templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      User: {{ user.username }}
+    </li>
+    <li>
+      <a href="{% url "logout" %}">Log out</a>
+    </li>
+  {% else %}  
+    <li><a href="{% url "login" %}">Log in</a></li>
+  {% endif %}
+  <br>
```

Comment:
Consider using CSS for spacing instead of `<br>` tags to maintain a cleaner HTML structure and better control over styling.